### PR TITLE
pm2-with-nodejs

### PR DESCRIPTION
**Pull request by HARITJOSHI1**
### Article checklist

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**
      
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**

Issue from #2434

### CODE REVIEW

**Comment by HARITJOSHI1** [comment]
@srishilesh sir only check the latest commit that is where my main article is

**Comment by HARITJOSHI1** [comment]
@srishilesh sir I think hero image is there

File: content/articles/pm2-with-nodejs/index.md
Comment (srishilesh) [inline content/articles/pm2-with-nodejs/index.md:267]: Convert sub headings with markdown `####`.

And, remove `&nbsp`

**Review by srishilesh — CHANGES_REQUESTED** [review]
Hey @HARITJOSHI1 ,

Thanks for letting me know. I've added the hero image label, just for my reference.

Great article. Very informative.

I've requested for a few changes, please let me know when done. We can then move on to the tech review.

Thanks.

**Comment by HARITJOSHI1** [comment]
> Hey @HARITJOSHI1 ,
> 
> Thanks for letting me know. I've added the hero image label, just for my reference.
> 
> Great article. Very informative.
> 
> I've requested for a few changes, please let me know when done. We can then move on to the tech review.
> 
> Thanks.

Yes sir I'm right on it

**Comment by HARITJOSHI1** [comment]
@srishilesh I have done the suggested changes please check them out.

**Comment by HARITJOSHI1** [comment]
@srishilesh I have done the suggested updates

**Comment by HARITJOSHI1** [comment]
@srishilesh I have done the suggested changes please check it out

**Review by srishilesh — APPROVED** [review]
@HARITJOSHI1 

Great article.

Moving this for the final review.

Thanks

**Comment by HARITJOSHI1** [comment]
@srishilesh what 's the status of my article?

**Comment by srishilesh** [comment]
> @srishilesh what 's the status of my article? 

@HARITJOSHI1 

The review is done from my side.

Please wait for the final review by Hector, before publishing this article.

Thanks

**Comment by HARITJOSHI1** [comment]
> > @srishilesh what 's the status of my article? 
> 
> @HARITJOSHI1 
> 
> The review is done from my side.
> 
> Please wait for the final review by Hector, before publishing this article.
> 
> Thanks

Okay sir I got it 😊

**Comment by hectorkambow** [comment]
[pm2-with-nodejs #2492.pdf](https://github.com/section-engineering-education/engineering-education/files/6639885/pm2-with-nodejs.2492.pdf)
Upon running your article through our 3rd party plagiarism checker it seemed to raise a many flags and the % was way higher than we typically accept.
Please see attached PDF - and revisit the article to ensure we are contributing wholly unique and original content.
Be sure to see our resources page to see more info on plagiarism and what is considered as such.

@HARITJOSHI1

**Comment by HARITJOSHI1** [comment]
@hectorkambow  I have updated my article according to your suggestions.

**Comment by HARITJOSHI1** [comment]
@hectorkambow status of the article?

**Comment by hectorkambow** [comment]
nice work @HARITJOSHI1 

Please go over my final edits to see how future articles may be improved. 

Cc @srishilesh  - please do the same.

**Comment by HARITJOSHI1** [comment]
> nice work @HARITJOSHI1 
> 
> Please go over my final edits to see how future articles may be improved. 
> 
> Cc @srishilesh  - please do the same. 

Thanks Hector 😊

**Pull request merged**
